.github/workflows/ci_pr.yaml

```
@@ -54,6 +55,16 @@ jobs:
         run: |
           echo "max_parallel=${{ env.max_parallel }}" >> $GITHUB_OUTPUT
 
+      - id: set_branch_name
+        name: set_branch_name
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:6:36: Double quote to prevent globbing and word splitting

(shellcheck)